src/components/itemContextMenu.js

```diff
@@ -589,9 +614,16 @@ function play(item, resume, queue, queueNext) {
             serverId: item.ServerId
         });
     } else {
+        const sortParentId = item.IsFolder ? ('items-' + item.Id) : libraryMenu.getTopParentId() + '-' + getSettingsKey(item);
+        const sortValues = userSettings.getSortValues(sortParentId);
```

Comment:
For now, I think we can leave just `Folder` (as in `shortcuts.js`) so they work the same way. And `Folder` requires `getSortValuesLegacy` anyway.

Also, `getTopParentId` is only valid for libraries and is empty for lists. So we should probably use `item.ParentId` (as again in `shortcuts.js`), and if it is a folder, we will get its sort options.
```suggestion
        const sortParentId = 'items-' + (item.IsFolder ? item.Id : item.ParentId) + '-Folder';
        const sortValues = userSettings.getSortValuesLegacy(sortParentId);
```